.github/workflows/pr.yml

```diff
@@ -0,0 +1,16 @@
+name: PR
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+concurrency: adminusers-pr
```

Comment:
Do we want this concurrency? Having it means that only a single PR will be able to run tests at once which might be quite painful for developers

Reply:
might be. Dependabot PRs open at once which might use loads of resources. We can remove it if it causes any issues for developers but it is very rare

Reply:
On the plus side, the resources are all GitHubs, not ours. Concourse running the e2e tests is hardly doing anything since it runs in CodeBuild, so it's mostly just polling waiting for the CodeBuild to finish.

We never had concurrency on the PR tests before and I'm quite reluctant for us to introduce it since it's an extra level of complexity, and I think an unnecessary slowing down of testing.

Reply:
👍🏼 removed